docs/content/utilities/documentation/moose_flavored_markdown.md

```diff
@@ -300,3 +302,22 @@ Currently this will only work with Civet CI services.
 ```
 !!! note
     Be sure to follow your !buildstatus extension with an empty new line.
+
+
+---
+
+## Bibliographies
+
+It is possible to include citations using latex commands, the following commands are supported within the markdown.
+
+* `\cite{slaughter2015continuous}`: \cite{slaughter2015continuous}
+* `\citet{wang2014diffusion}`: \citet{wang2014diffusion}
+* `\citep{gaston2015physics}`: \citep{gaston2015physics}
+
+The bibliography style may be set within a page using the latex command
+`\bibliographystyle{unsrt}`. Three styles are currently available: 'unsrt', 'plain', 'alpha', and 'unsrtalpha'.
+
+The references are displayed by using the latex `\bibliography{docs/moose.bib}` command. This command accepts a comma separated list of bibtex files (*.bib) to use to build citations and references. The files specified in this list must be given as a relative path to the root directory (e.g., `~/projects/moose`) of the repository.
+
+\bibliographystyle{unsrt}
+\bibliography{docs/moose.bib}
```

Comment:
It looks like you can place your bibtex files where ever you would like. So if you wanted to place them all in one location, it will support that. For this example, it looks like @aeslaughter is just using the docs directory.